lesson03/assignment/css/style.scss

```diff
@@ -1,6 +1,65 @@
-// Import partials here
-@import 'base.normalize';
-@import 'settings.variables';
-@import 'settings.responsive';
-@import 'components.content';
+// Uses 2 partials compiled with a imports in main.scss
+@import 'css/_variables';
+@import 'css/_base';
+@import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
+// 2 Declares and uses mixin properly
+@mixin flex-display($direction: row, $space: flex-start, $align: center) {
+  display: flex;
+  justify-content: $space;
+  align-items: $align;
+}
 
+/*--------------------------------------------------------------
+# Elements
+--------------------------------------------------------------*/
+a {
+  text-decoration: none;
+  color: $text-grey;
+}
+/*--------------------------------------------------------------
+# Main
+--------------------------------------------------------------*/
+.main {
+  @include flex-display($space: space-around);
+  border-style: solid;
+  border-color: $background-grey;
+  border-width: 2em;
+  color: $text-grey;
+  width: 50em;
+  padding: 1em 0;
```

Comment:
`rem` should be your default sizing unit. Only use `em` when you have a very specific need for it, and in that case, add some comments. More info on the difference here: https://sass-lang.com/documentation/at-rules/use